packages/builder/src/components/database/ModelDataTable/LinkedRecord.svelte

```diff
@@ -0,0 +1,92 @@
+<script>
+  import { onMount } from "svelte"
+  import { fade } from "svelte/transition"
+  import { backendUiStore } from "builderStore"
+  import api from "builderStore/api"
+
+  export let ids = []
+  export let header
+
+  let records = []
+  let open = false
+
+  async function fetchRecords() {
+    const FETCH_RECORDS_URL = `/api/${$backendUiStore.selectedDatabase._id}/records/search`
```

Comment:
ooo. I realise that this is the easiest path. and will probably be fine for now.

However, this has the potential to blast the server/db .

I'll say it... here comes... I think we should store the `_id` and `name` (where "name" in this case is the models "primary field")

so... a record with a linked record would look like

```
{ surname: "johnston", forename: "joe", company: { _id: "1234abcd", name: "Budibase" } 
```

controversial and complicated i know... but i thin we want to avoid this `N+1` lookup

Reply:
Yeah I’m not thrilled about the N+1 either. I tried to make a view for it so that you can query a record and it’s linked records out of couch, but couch doesn’t support nested trees, so it just gives you a flat array which won’t really work if you have several linked record types. 

I think that we might need to just link things based on a “primary key” and think about how to model it in a better way